source/jobs/creating-jobs.txt

```diff
@@ -3,3 +3,148 @@
 =================
 Create a Sync Job
 =================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Create new sync jobs from the :guilabel:`Data Migration` tab.
+
+About This Task
+---------------
+
+The URI credentials you provide when creating a sync job do not 
+need to be the same as the credentials used when creating your project.
+
+Before you Begin
+----------------
+
+- :ref:`Create one or more mapping rules <rm-create-mapping-rules>` in your 
+  Relational Migrator project.
+- Prepare a :ref:`MongoDB URI <rm-mongodb-connection-string>` and
+  credentials that have read/write permissions on your destination database.
+
+Steps
+-----
+
+#. On the :guilabel:`Data Migration` tab, click :guilabel:`Create Sync Job`. 
+   Relational Migrator only runs one sync job at a time. If a sync job is in progress, this button is disabled.
+
+#. On the :guilabel:`Connect Source DB` form, enter the connection details 
+   to create the JDBC URI for your relational database. 
+
+   2a. Select a database type from the :guilabel:`Database type` drop-down.
+
+   2b. Enter a host IP or DNS name in the :guilabel:`Host` text box. 
+
+   2c. Enter a port number in the :guilabel:`Port` text box. 
+
+   2d. Enter a database name in the :guilabel:`Database` text field.
+
+   Depending on your relational database, this behavior varies:
+
+   .. list-table::
+      :header-rows: 1
+
+      * - Database Type
+        - Behavior
+      * - Oracle 
+        - You must enter a database name and a :guilabel:`Service ID` or :guilabel:`SID`.
+      * - SQL Server
+        - Enter a database name, or leave blank to load all databases.
+      * - MySQL 
+        - Enter a database name, or leave blank to load all databases.
+      * - Postgres 
+        - Leaving database name blank loads schemas from the default database.
+
+   2e. Enter a username in the :guilabel:`Username` text box. 
+
+   2f. Enter a password in the :guilabel:`Password` text box. 
+
+   2g. (Optional) Click the :guilabel:`Save a password for this session` checkbox.
+
+   2h. Click the :guilabel:`SSL` toggle switch to enable or disable SSL and select an SSL mode.
+
+   2i. Click :guilabel:`Connect`.
+
+   .. note::
+
+      Connection and SSL details depend on the database type you are connecting to. 
+      In addition to the generic connection properties listed above, you may also need
+      to select a :guilabel:`Identifier` for Oracle and :guilabel:`Authentication`
+      for SQL Server.
+
+   If you want to specify the JDBC URI manually, click the 
+   :guilabel:`Enter URI manually` toggle switch on the :guilabel:`Connect SourceDB` 
+   form. For details, see :ref:`rm-relational-database-connection-strings`
```

Comment:
I think that I removed the note annotation awhile back because I didnt like having back to back note directives. It also seemed like more than an aside and somewhat more likely users will use the manual method so thats why I broke it back into the standard copy.